### ISSUE TITLE
Wrong executable name for DS9 in OSX

### ISSUE THREAD

**Issue by npirzkal** [ericmandel/pyds9#105]
pyds9 looks for /Applications/SAO ImageDS9.app when running in OSX but the current versions of DS9 are now installed as /Applications/SAOImageDS9.app (no space). This causes pyds9 to complain that it cannot find any executable and it does not start  ds9 automatically. Removing the space on line 75 of pyds9.py fixed it for me. This is using SAOImageDS9 v. 8.2.1

**Comment by ericmandel** [comment]
The version of pyds9.py in github looks for SAOImageDS9.app as well as the
older filename with spaces. Can you please try upgrading to the guthub
version?

**Comment by npirzkal** [comment]
Thanks. I just did a pip install so I assumed it was not patched :-)


N

**Comment by ericmandel** [comment]
Agreed that it would be great to have pip updated, but we have no python expert to do that sort of work.

**Comment by npirzkal** [comment]
The way I do it is pretty straight forward:

python setup.py sdist bdist_wheel
twine upload dist/*

You will need twine installed but I do this with a simple "pip install twine"
You can register and create an account at https://pypi.org/project/twine/

I hope that helps.

**Comment by ericmandel** [comment]
Thanks, if I get time in the next month, I'll look into it ... but
unfortunately, this really is a question of support that can't be fixed ad
hoc.
